src/state/Components/helpers.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Set value on given path. If value is undefined, then delete property on path.
+ * @param state {Object}
+ * @param path {Array}
+ * @param value {*}
+ * @return {Object}
+ */
+function setHelper(state, path, value) {
+	let remainingPath = [...path];
```

Comment:
What if path is undefined?

Reply:
fixed: f5621ef73fa5e2dd617500cf4fe70c44803d1042